instance/tests/models/test_instance.py

```diff
@@ -336,10 +344,12 @@ def test_run_playbook(self, mock_open_repo, mock_run_playbook, mock_inventory, m
         ), mock_run_playbook.mock_calls)
 
 
+#pylint: disable=too-many-public-methods
```

Comment:
@omarkhan Would it be worth splitting this `TestCase` class into several ones, rather than silencing the warning?